### PR TITLE
fixs rid in testing docs

### PR DESCRIPTION
**Pull request by jscyo**
## Summary of change
Change `rid` value to `session` in testing with postman docs for emailpassword, thirdparty and thirdpartyemailpassword recipies

## Related issues
#85 

## Remaining TODOs for this PR
none

### CODE REVIEW

**Comment by rishabhpoddar** [comment]
Use variable feature and copy docs for thirdpartyemailpassword and emailpassword docs

File: v2/src/plugins/markdownVariables.json [hunk 1]
```diff
@@ -20,14 +20,18 @@
...
     },
     "thirdparty": {
         "docsLinkRecipeName": "thirdparty",
         "codeImportRecipeName": "thirdparty"
     },
     "emailpassword": {
         "docsLinkRecipeName": "emailpassword",
-        "codeImportRecipeName": "emailpassword"
+        "codeImportRecipeName": "emailpassword",
+        "ridRecipeName": "emailpassword",
```
Comment (rishabhpoddar) [inline v2/src/plugins/markdownVariables.json:34]: Call this `rid`

File: v2/src/plugins/markdownVariables.json [hunk 2]
```diff
@@ -20,14 +20,18 @@
...
+        "recipeName": "EmailPassword"
```
Comment (rishabhpoddar) [inline v2/src/plugins/markdownVariables.json:35]: Call this `recipeNameCapitalLetters`

File: v2/emailpassword/testing/testing-with-postman.mdx [hunk 3]
```diff
@@ -60,34 +66,34 @@ app.post("/change-user-data", Session.verifySession(), async (req, res) => {
...
 - If you have the `antiCsrf` attribute set to `VIA_TOKEN` in your backend SuperTokens config, then, in the Postman Header tab, set the key as `anti-csrf` and value as the `anti-csrf` token retrieved from the login response.
-- On a successful response, the response body will contain the `user` object
+- On a successful response, the response body will contain user data
 
 :::important
 By default, for `GET` APIs, you don't need to provide the `anti-csrf` request header as anti-CSRF checks are only done in non-`GET` APIs
 :::
 
-<img width="600px" src="/img/emailpassword/st-ep-change-user.png" />
+<img width="600px" src="/img/change-user.png" />
```
Comment (rishabhpoddar) [inline v2/emailpassword/testing/testing-with-postman.mdx:83]: You can use variables in the src link as well.. So use those and have different images as expected.

**Review by rishabhpoddar — CHANGES_REQUESTED** [review]

File: v2/src/plugins/markdownVariables.json (hunk 1 above)
Comment (jscyo) [reply v2/src/plugins/markdownVariables.json]: done

File: v2/src/plugins/markdownVariables.json (hunk 2 above)
Comment (jscyo) [reply v2/src/plugins/markdownVariables.json]: done

File: v2/emailpassword/testing/testing-with-postman.mdx (hunk 3 above)
Comment (jscyo) [reply v2/emailpassword/testing/testing-with-postman.mdx]: done

**Pull request merged**